scripts/revng_merge_dynamic/__main__.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import sys
+
+from .log import set_verbose
+from .merge_dynamic import merge_dynamic
+
+parser = argparse.ArgumentParser(
```

Comment:
Why global? Can we move it in `main`?

Reply:
I've pretty much always seen the argument parser as a global variable in all the codebases, in this case since it has no side effects and nobody has any reason to import `__main__.py` anyway there's no downside I can think of (note: all the other modules do not have `parser` in their scope).
But doing it as you sugges has no downsides as well, I guess it's ultimately up to personal preference/conventions so I'll move it into `main`, no problem

Reply:
It's a way to prevent people from importing it. In general, let's avoid global variables and let's try to keep the scope of things as short of possible.